doc/content/htgr/httf/lower_plenum_cfd.md

```diff
@@ -0,0 +1,129 @@
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat2() end=subroutine usrdat3() include-end=False
+
+The $k-\tau$ URANS model is specified in the following code block
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat3() end=subroutine turb_in(wd,tke,tau) include-end=False
+
+The specific inlet temperature and mass flow rates are implemented in +userbc+ with the non-dimensionalized values
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=if(id_face.eq.3) end=endif include-end=True
+
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
+A characteristics based time-stepping has been used in the simulations to avoid the limitations imposed by the Courant-Friedrichs-Lewy
+(CFL) number due to the explicit treatment of the non-linear convection
+term. This treatment is necessary due to the existence of small local mesh cells created during the unstructured meshing and tet-to-hex conversion. 
+A maximum CFL number of 1.75 has been reached which corresponds to an average time step size of $2.5\times 10^{-5}$.
+A total simulation time of 5.6 is achieved. 
+Due to the nature of unsteady RANS turbulence model, only a quasi-steady state can be reached. Velocity fluctuations are observed when helium flow enters the lower plenum and passes through the LP posts.
+
+Snapshots of velocity and temperature fields from quasi-steady state are shown in [lp_vel] and [lp_temp]
+at the lower plenum plane ($y=-0.1 m$). Significant fluctuations are observed for both velocity and temperature. 
+
+!media httf/lower_plenum_cfd/unsteady_vel.png
+       style=width:60%
+       id=lp_vel
+       caption=The quasi-steady-state velocity field (non-dimensional) in the lower plenum.
+
+!media httf/lower_plenum_cfd/T_at_time_5.6.png
+       style=width:60%
+       id=lp_temp
+       caption=The quasi-steady-state temperature field (non-dimensional) in the lower plenum.
+
+In addition to the instantaneous solutions, a time averaging analysis is also conducted for simulation results from 3.8 to 5.6 time units.
+[Uavg] and [Tavg] illustrate the smooth profiles of time averaged velocity and temperature field respectively. 
+Large velocity difference is noticed due to the geometric constraints in lower plenum. On the contrary, the temperature has a more uniform distribution for the most part of lower plenum. 
+
+!media httf/lower_plenum_cfd/Uavg.png
+       style=width:60%
+       id=Uavg
+       caption=Time-averaged velocity field (non-dimensional) in the lower plenum.
+
+!media httf/lower_plenum_cfd/Tavg.png
+       style=width:60%
+       id=Tavg
+       caption=Time-averaged temperature field (non-dimensional) in the lower plenum.
```

Comment:
do you want to add a paragraph to give perspective on this? like where you're going